Assets/Scripts/UI/LobbyCodeInputTextField.cs

```diff
@@ -0,0 +1,18 @@
+using UnityEngine;
+
+namespace UI
+{
+    public class LobbyCodeInputTextField : MonoBehaviour, ITextInputFieldReader
+    {
+        public JoinLobbyButton _joinLobbyButton; 
+        
+        public void ValueChanged(string value)
+        {
+        }
+
+        public void EditEnded(string value)
```

Comment:
I am slightly uncertain as to the purpose of using this script and this interface; I'm curious why we are not handling this functionality somewhere more easily accessible, at the top level of our hierarchy, and adding a method directly to our InputField's OnEndEdit event, though this might be more transparent when viewing the entire codebase. I understand the desire to observe the Single Object Dependency principle as much as possible by splitting things off into their own classes, but when our solution for doing so requires that we place MonoBehaviours deep into our transform hierarchy, there is probably a more elegant solution.